pycti/entities/opencti_report.py

```diff
@@ -814,7 +816,12 @@ def remove_stix_object_or_stix_relationship(self, **kwargs):
             query = """
                mutation ReportEditRelationDelete($id: ID!, $toId: StixRef!, $relationship_type: String!) {
                    reportEdit(id: $id) {
-                        relationDelete(toId: $toId, relationship_type: $relationship_type) {
+                        relationDelete(
+                            toId: $toId, 
+                            relationship_type: $relationship_type, 
+                            commitMessage: $commitMessage, 
+                            references: $references
```

Comment:
same here : `$commitMessage` and `$references` need to be declared in `ReportEditRelationDelete` mutation